packages/components/checkbox/src/CheckboxLabel.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+const defaultCheckboxVariants = {
+  disabled: false,
+}
+
+export const checkboxLabelStyles = cva(['flex', 'items-center', 'gap-md', 'text-body-1'], {
+  variants: {
+    disabled: {
+      true: ['text-neutral/dim-2', 'cursor-not-allowed'],
+      false: ['cursor-pointer'],
+    },
+  },
+  defaultVariants: defaultCheckboxVariants,
```

Comment:
```suggestion
  defaultVariants: {
    disabled: false,
  },
```

Reply:
I always split it because when we start adding variants with strings instead of boolean, doing it directly in the `cva()` argument generates eslint errors (invalid TW classNames)